java-util/pom.xml

```diff
@@ -85,6 +85,11 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <groupId>com.yahoo.datasketches</groupId>
```

Comment:
@leerho it is already done since https://github.com/DataSketches/sketches-core/commit/92e67a9ab1d96df18b9676fdf7af675033b646d8. and the dependency is already `com.yahoo.datasketches:memory`, that I missed when wrote my previous comments in this small thread.